src/mappings/_utils.ts

```diff
@@ -155,13 +153,12 @@ export function divideByTenToTheNth(amount: bigint, n: number): number {
 }
 
 export async function symbolFromCurrency(currency: Currency): Promise<string> {
```

Comment:
Just a thought: maybe we could rename this to `tickerFromCurrency` and use word ticker instead of symbol everywhere so that nomenclature is consistent across the repos, what do you think?

Reply:
That's a good shout, I like it. But it would be a breaking change - also for the UI, I suspect. 
So I raised that in a separate issue: https://github.com/interlay/interbtc-squid/issues/138